remme/rpc_api/transaction.py

```diff
@@ -93,6 +98,23 @@
             message='Failed to parse transaction proto'
         )
 
+    try:
+        tr_head_pb = TransactionHeader()
+        tr_head_pb.ParseFromString(tr_pb.header)
+    except DecodeError:
+        raise RpcGenericServerDefinedError(
+            error_code=-32050,
+            message='Failed to parse transaction head proto'
+        )
+
+    prefix, public_key = tr_head_pb.signer_public_key[:2], \
```

Comment:
Why would we need this check here? Actually if a format do not conform to this check it will fail earlier with the `DecodeError `